internal/server/requests.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import(
+	"net/http"
+	"fmt"
+	"errors"
+	"log"
```

Comment:
According to [acceptance criteria](https://github.com/sgrzywna/golang-academy/tree/main/workshop_1#acceptance-criteria) you should use logging as described [here](https://github.com/google/exposure-notifications-server/tree/main/pkg/logging).